src/components/Button/index.examples.js

```diff
@@ -8,14 +8,14 @@ const standardButtonExamples = [
   {type: 'primary', children: 'Primary'},
   {type: 'success', children: 'Success'},
   {type: 'warning', children: 'Warning'},
-  {type: 'danger', children: 'Danger'}
+  {type: 'danger', children: 'Danger'},
+  {type: 'orange', children: 'Orange'},
+  {type: 'turquoise', children: 'turquoise'}
```

Comment:
@trevordmiller  I'm strictly referring to the core button, not contextual implementation.